go/teams/treeloader.go

```diff
@@ -0,0 +1,245 @@
+package teams
+
+import (
+	"fmt"
+	"sync/atomic"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"golang.org/x/sync/errgroup"
+)
+
+type NodePosition int
+
+const (
+	NodePositionTarget   = 0
+	NodePositionAncestor = 1
+	NodePositionChild    = 2
+)
+
+// LoadTeamTreeMemberships takes a teamID and a username and asynchronously spawns notifications
```

Comment:
Not sure whether to put these comments in the AVDL or here.

Reply:
i like here better